### DISCUSSION TITLE
Trouble checking out saleor 3.0 with saleor-platform

### DISCUSSION THREAD

**Discussion by vidhyau-twbs-eu** [saleor/saleor#7917] [Q&A]
Can anyone share steps to run saleor 3.0 (saleor/dashboard/storefront) with the platform ?
I am having difficulty. I want to get the multi currency working with saleor urgently.

Can someone help?

**Comment by karolkielecki** [comment]
@vidhyau-twbs-eu Could you please share the steps you've taken so far? If you've installed node.js, go to the catalog and run `npm install` and `npm start` --that's how you will run it outside docker.

**Reply by vidhyau-twbs-eu** [reply]
1. Used the stable saleor-platform
2. checked out latest 3.0 tags for each submodule saleor/saleor-dashboard/saleor-storefront
3. tried running docker compose build and docker compose up
4. Saleor - issue running graphql - Multiple channels are active error shows up, and i had to do a manual database update to set is_active false to to channel with id=2 (Then this was fixed)
5. Saleor-Dashboard and saleor-storefront not starting because of package.json referring macaw-ui with commit version. 
   "@saleor/macaw-ui": "github:mirumee/macaw-ui#03d63cd",
   
   NPM install does not pull this dependeny, and hence module not found error.

**Reply by karolkielecki** [reply]
Ok, thanks! I'll pass it to the team and we'll check it out.

**Reply by karolkielecki** [reply]
Indeed, it seems there is an issue with the dashboard and storefront using the `saleor-platform` repo. The team has been informed about it. If I have any news, I'll let you know. In the meantime, if you want to use 3.0 please use the following repos: 

- https://github.com/mirumee/saleor-storefront/releases
- https://github.com/mirumee/saleor/releases
- https://github.com/mirumee/saleor-dashboard/releases

**Reply by vidhyau-twbs-eu** [reply]
I will wait for it, More issues i am finding, which i will post in another thread
